base.cpp:82:16: error: ‘invalid_argument’ is not a member of ‘std’

Trying to build Lydia on Ubuntu 21.04, gcc 10.3.0

`lydia/lib/src/logic/ltlf/base.cpp:82:16: error: ‘invalid_argument’ is not a member of ‘std’
   82 |     throw std::invalid_argument("LTLfAnd formula: arguments must be > 1");
      |                ^~~~~~~~~~~~~~~~
`

Adding `#include <stdexcept>` to base.cpp would fix the bug.

--- a/lib/src/logic/ltlf/base.cpp
+++ b/lib/src/logic/ltlf/base.cpp
@@ -18,6 +18,7 @@
 #include <cassert>
 #include <lydia/logic/ltlf/base.hpp>
 #include <lydia/utils/compare.hpp>
+#include <stdexcept>
 #include <utility>
 
 namespace whitemech::lydia {

